demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -117,6 +117,21 @@ def is_docker_image_same_as_yml(self) -> bool:
 
         return True
 
+    def is_breaking_change(self) -> bool:
```

Comment:
```suggestion
    error_codes('<ADD_HERE_THE_ERROR_CODE>')
    def is_breaking_change(self) -> bool:
```
please take a look at existing validations and see what you need to add - e.g. lines 113-115
do we need to create another error code or use an existing one?